main.py

```diff
@@ -17,7 +17,7 @@ def saveJsonTime():
 
 def initStartProgram(): 
     for i in range(9):
```

Comment:
```suggestion
    for _ in range(9):
```

파이썬에선 사용하지 않는 변수가 있으면 `_`를 쓴다고 알고있습니다